python/sedona/spark/geopandas/geoseries.py

```diff
@@ -1123,6 +1123,28 @@ def union_all(self, method="unary", grid_size=None) -> BaseGeometry:
         geom = ps_series.iloc[0]
         return geom
 
+    def intersection_all(self, method="unary", grid_size=None) -> BaseGeometry:
+        if grid_size is not None:
+            raise NotImplementedError("Sedona does not support the grid_size argument")
+        if method != "unary":
+            import warnings
+
+            warnings.warn(
+                f"Sedona does not support manually specifying different intersection methods. Ignoring non-default method argument of {method}"
+            )
```

Comment:
```suggestion
    def intersection_all(self) -> BaseGeometry:
```

`intersection_all()` in geopandas doesn't have these parameters. See the geopandas docs [here](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.intersection_all.html)